test/config.sh

```diff
@@ -111,6 +111,14 @@ start_container_with_dns() {
     weave_on $host run --with-dns "$@" -t $DNS_IMAGE /bin/sh
 }
 
+rm_containers() {
+    host=$1
+    shift
+    if [ $# -gt 0 ] ; then
+        docker_on $host rm -f "$@" >/dev/null 2>&1
+    fi
+}
```

Comment:
`[ $# -eq 0 ] || docker_on $host rm -f "$@" >/dev/null 2>&1`

Why the `2>&1`? Given that this whole thing fails when the rm fails, it seems odd that we'd swallow the error message. I know we had that indirection in `start_suite`, but I don't know why. Does `rm` produce output on stderr in some success cases?